tools/reaper/internal/internal_test.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 func TestServeAndClose(t *testing.T) {
-	conn, err := net.Listen("tcp", ":0")
```

Comment:
Just curious, does it use 127.0.0.1 instead without specifying locahost?

Reply:
No it uses 0.0.0.0 which means to serve everyone.